packages/common/src/index.ts

```diff
@@ -125,18 +140,37 @@ export default class Common {
     }
   }
 
+  /**
+   * Sets a new hardfork based on the block number provided
+   * @param blockNumber
+   * @returns The name of the HF set
+   */
+  setHardforkByBlockNumber(blockNumber: number): string {
```

Comment:
I think we should also add the possibility that `blockNumber` is `null`. For instance, in all testnet chain configs the DAO support is in the list of supported forks, but the "activation block number" is `null`, i.e. it is never activated (the alternative is to use a super high block number but this is not clean).

Reply:
Never mind got the semantics of this function wrong.